### PR TITLE
Bc/misc small cleanup

### PR DESCRIPTION
**Pull request by bryancunningham-okta**
[OKTA-710526](https://oktainc.atlassian.net/browse/OKTA-710526)
[OKTA-705411](https://oktainc.atlassian.net/browse/OKTA-705411)
[OKTA-690257](https://oktainc.atlassian.net/browse/OKTA-690257)
[OKTA-708296](https://oktainc.atlassian.net/browse/OKTA-708296)

## Testing & Screenshots

- [ ] I have confirmed this change with my designer and the Odyssey Design Team.

### CODE REVIEW

**Review by benlister-okta — DISMISSED** [review]

**Review by chrispulsinelli-okta — DISMISSED** [review]

**Review by jordankoschei-okta — DISMISSED** [review]

File: packages/odyssey-react-mui/src/FieldError.tsx [hunk 1]
```diff
@@ -36,7 +35,13 @@ const FieldError = ({
   const { t } = useTranslation();
 
   return (
-    <FormHelperText data-se={testId} error id={id} translate={translate}>
+    <FormHelperText
+      component="div"
```
Comment (bryancunningham-okta) [inline packages/odyssey-react-mui/src/FieldError.tsx:39]: Had to change to `div` to fix invalid nesting of `div` inside `p`

File: packages/odyssey-react-mui/src/ScreenReaderText.tsx [hunk 2]
```diff
@@ -20,6 +20,7 @@ export type ScreenReaderTextProps = {
    * The visually-hidden text.
    */
   children: ReactNode;
+  id?: string;
```
Comment (bryancunningham-okta) [inline packages/odyssey-react-mui/src/ScreenReaderText.tsx:23]: This change is left over from my initial effort to move the screen reader text out of the actual elements and use `aria-describedby`. Figured we will eventually make that change so, may as well leave this in

File: packages/odyssey-react-mui/src/FieldError.tsx (hunk 1 above)
Comment (jordankoschei-okta) [reply packages/odyssey-react-mui/src/FieldError.tsx:39]: Good catch 👍

File: packages/odyssey-storybook/src/components/odyssey-mui/Tabs/Tabs.stories.tsx [hunk 3]
```diff
@@ -111,15 +111,18 @@ const storybookMeta: Meta<TabsProps & TabItemProps> = {
...
+        type: "select",
+      },
+      options: ["10", "20", "30", "40", "50", "60", "70", "80", "90", "100"],
       description:
-        "The limit at which the badge will show '`{notificationCountMax}`+'. A number between 0-999",
+        "The limit at which the badge will show `{notificationCountMax} +`. Can be increments of 10, up to 100",
       table: {
         type: {
-          summary: "number",
+          summary: "10 | 20 | 30 | 40 | 50 | 60 | 70 | 80 | 90 | 100",
```
Comment (jordankoschei-okta) [inline packages/odyssey-storybook/src/components/odyssey-mui/Tabs/Tabs.stories.tsx:122]: Curious why we decided on increments of 10 here — will some features really want the + to start at 20 vs 30?

**Review by jordankoschei-okta — APPROVED** [review]

File: packages/odyssey-storybook/src/components/odyssey-mui/Tabs/Tabs.stories.tsx (hunk 3 above)
Comment (bryancunningham-okta) [reply packages/odyssey-storybook/src/components/odyssey-mui/Tabs/Tabs.stories.tsx:122]: @jordankoschei-okta Mostly to try and enforce _some_ consistency. So folks don't set the max at something like `37` or whatever. We thought about just making it `10` but, I know for sure there is a use case that uses `20`. So, we just said increments of 10, up to 100 would probably work.

**Review by KevinGhadyani-Okta — APPROVED** [review]

**Pull request merged**